pkg/packagekit/updates.jsx

```diff
@@ -1472,23 +1473,23 @@ class OsUpdates extends React.Component {
             });
 
             return (
-                <>
+                <Stack hasGutter>
                     <EmptyStatePanel title={ STATE_HEADINGS[this.state.state] }
-                                     icon={ ExclamationCircleIcon }
-                                     paragraph={
-                                         <TextContent>
-                                             <Text component={TextVariants.p}>
-                                                 {this.state.errorMessages
-                                                         .filter((m, index) => index == 0 || m != this.state.errorMessages[index - 1])
-                                                         .map(m => <span key={m}>{m}</span>)}
-                                             </Text>
-                                             <Text component={TextVariants.p}>
-                                                 {_("Please reload the page after resolving the issue.")}
-                                             </Text>
-                                         </TextContent>
-                                     }
+                                    icon={ ExclamationCircleIcon }
+                                    paragraph={
+                                        <TextContent>
+                                            <Text component={TextVariants.p}>
+                                                {_("Please resolve the issue and reload this page.")}
+                                            </Text>
+                                        </TextContent>
+                                    }
                     />
-                </>
+                    <CodeBlockCode className='error-log'>
```

Comment:
You're using `CodeBlockCode`, but it isn't actually making this a code block. The font isn't correct.

I'm not sure why it's not pulling in the font correctly. A guess: Is the import not right? That would also explain why it's not styled at all.

![Screenshot from 2023-04-04 12-54-57](https://user-images.githubusercontent.com/10246/229771950-37ff4bbc-f51a-4795-8b3c-f9af130506f2.png)

Reply:
Yes this was concerning me as well.

Reply:
Even after using CodeBlock, the --pf-c-code-block__code--FontFamily is undefined

Reply:
The problem is that you 're using codeblockcode without wrapping it with codeblock. The styles are all from codeblock, not codeblockcode.

Reply:
Oh? I guess you're testing it locally. PF often sets variables like that in a topmost element and then inherits the variable from children elements.

Looking at the React example on PatternFly's website, I see this:

![image](https://user-images.githubusercontent.com/10246/229793300-6123e8e1-56a9-49dc-8058-1b326ab71f9a.png)

And that shows me that `--pf-c-code-block__pre--FontFamily` is set in `pf-c-code-block`.

So I'm not sure why it's not working when you're using something like:

```jsx
<CodeBlock>
  <CodeBlockCode>
    ERRORS GO HERE
  </CodeBlockCode>
</CodeBlock>
```
(...With the import I suggested elsewhere in this PR that pulls in both CodeBlock and CodeBlockCode.)

@KKoukiou would probably know more.

Reply:
It's finally fixed. I didn't run test/image-prepare before testing.